src/components/PickupBox/PickupBox.module.css

```diff
@@ -131,3 +131,60 @@
 .image-corners {
   border-radius: 5%;
 }
+
+.image-functionality-wrapper {
+  display: flex;
+  width: 100%;
+}
+
+.image-functionality {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.approval-button {
+  float:left;
+  margin: 2% 2% 0% 0%;
+}
+
+.rejection-button {
+  float:left;
+  margin: 2% 2% 0% 0%;
+}
+
+.image-approved {
+  border: 1px solid green;
+}
+
+.image-rejected {
+  border: 1px solid red;
+}
+
+.approval-message {
+  color: green;
+  float:left;
+  padding: 1%;
+  width:100%;
+  text-align: left;
+  margin-left: 1%;
+}
+
+.rejection-message {
+  color: red;
+  float:left;
+  width:100%;
+  padding: 1%;
+  text-align: left;
+  margin-left: 1%;
+}
```

Comment:
These have a lot of repeated styling - it would be best to create another selector (e.g. .message) that contains all the common styling among these two selectors and then have .approval-message and .rejection-message only set the color.